DEVELOPMENT.md

```diff
@@ -92,8 +101,13 @@ By default KubeArchive listens to `Event`s in the `test` namespace.
     ```
 1. On a new terminal, use `curl` or your browser to perform a query:
     ```bash
-    curl localhost:8081/apis/apps/v1/deployments
+    curl -H "Authorization: Bearer $(kubectl create token kubearchive-privileged-user)" \
+   localhost:8081/apis/apps/v1/deployments
     ```
+   **NOTE**: To test with an unauthorized user replace `kubearchive-privileged-user` with 
+   an unprivileged service account like `default`.
```

Comment:
I don't think this belongs here, as we are not guiding the contributors into manual testing, but just teaching them how to generate traffic and do basic a basic GET on the API. If we want to add some manual testing section I would do that in the normal documentation (which is not added to the repository yet).

Reply:
I can change the perspective and tell this information in the "known issues" section. 
I want people to know that if they don't use a token from a privileged service account, they won't get the expected output.